data/default_release_images.json

```diff
@@ -72,18 +72,18 @@
         "cpu_architectures": [
             "x86_64"
         ],
-        "url": "quay.io/openshift-release-dev/ocp-release:4.12.0-ec.2-x86_64",
+        "url": "quay.io/openshift-release-dev/ocp-release:4.11.4-x86_64",
```

Comment:
weird why it should do that
I'll patch this PR for now and later on figure out why the bump script did it